### PR TITLE
Task/3644 login btn on dcl navbar

### PR DESCRIPTION
**Pull request by tiwariav**

### CODE REVIEW

**Comment by verysanghee** [comment]
tested on dev, looks good. Can you please review? @pcoder

File: datacenterlight/templates/datacenterlight/includes/_navbar.html [hunk 1]
```diff
@@ -34,6 +34,17 @@
...
+                {% if not request.user.is_authenticated %}
+                    <li>
+                        <a href="{% url 'hosting:login' %}">{% trans "Login" %}&nbsp;&nbsp;<span class="fa fa-sign-in"></span></a>
+                    </li>
+                {% else %}
+                    <li>
+                        <a href="{% url 'hosting:dashboard' %}">{% trans "Dashboard" %}</a>
+                    </li>
+                {% endif %}
+                {% if request.path == '/en-us/datacenterlight/' or request.path == '/de/datacenterlight/' %}
```
Comment (pcoder) [inline datacenterlight/templates/datacenterlight/includes/_navbar.html]: Do we intend to do something here? If not, it can be removed I guess.

**Comment by pcoder** [comment]
Otherwise, it looks good to be merged.

File: datacenterlight/templates/datacenterlight/includes/_navbar.html (hunk 1 above)
Comment (tiwariav) [reply datacenterlight/templates/datacenterlight/includes/_navbar.html]: Yes, should be removed. 👍

**Pull request merged**